charts/infra-apps/examples/thanos.yaml

```diff
@@ -0,0 +1,5 @@
+thanos:
+  enabled: true
+  values:
```

Comment:
Can you add enough to this example so it shows how a tipical thanos deployment would look like?

Reply:
932e735